.github/workflows/cleanup-old-deployments.yml

```
@@ -0,0 +1,17 @@
+name: Cleanup old deployments
+
+on:
+  workflow_dispatch:
+
+jobs:
+  cleanup-old-deployments:
```

One small request -- could we make clear in the name that it only cleans up in GH and not in k8s?

```suggestion
  cleanup-old-deployments-from-gh
```

---

Done in [90fcbcb](https://github.com/SwissDataScienceCenter/renku-ui/pull/3074/commits/90fcbcb7867b54e1eed38e731eb5999475c7b169).